### DISCUSSION TITLE
Validate regex manager input expressions

### DISCUSSION THREAD

**Discussion by zharinov** [renovatebot/renovate#22210] [Suggest an Idea]
**What would you like Renovate to be able to do?**

For example, check for unsupported capture groups.

**Did you already have any implementation ideas?**

**Comment by rarkins** [comment]
"Unsupported" capture groups can be reused in template fields though, so we can't merely warn if it's not one of our standard ones unless we then check for its use in the template fields to. Did anyone have any further ideas of what to validate or examples of wrong config?
